packages/os/bootstrap-containers@.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=bootstrap container %i
+Before=configured.target
+After=host-containerd.service
+
+[Service]
+Type=oneshot
+EnvironmentFile=/etc/bootstrap-containers/%i.env
+Environment=LOCAL_DIR=/local
+ExecStart=/usr/bin/host-ctr run \
+    --container-id='%i' \
```

Comment:
Are bootstrap containers allowed to have the same name as host containers?  If you're using the name as the container ID and they are allowed to have the same name, you won't actually be able to start both (host-ctr will see that a container already exists with the given ID and not create a new one).

Reply:
I updated the bootstrap-containers initialization and they now are created in the `bootstrap` namespace to prevent collisions.

Reply:
I changed the implementation to use prefixes instead of namespaces